src/GafferOSL/ShadingEngine.cpp

```diff
@@ -431,13 +526,158 @@ struct ThreadRenderState
 namespace
 {
 
+template< int WidthT >
+class GafferBatchedRendererServices : public OSL::BatchedRendererServices<WidthT>
+{
+	// To avoid having the specify the WidthT parameter
```

Comment:
Hmm, I'm not quite sure why there isn't an easy way to inherit the `using`s from the parent class.  I can make it work without the macro, but I have to say: `typename OSL::BatchedRendererServices<WidthT>::BatchedShaderGlobals` instead of just `BatchedShaderGlobals`.  Getting into some fairly black magic template resolution stuff, so I'll probably just keep the macro and update the comment.

Reply:
Addressed in 8c84600